client/state/action-types.js

```diff
@@ -1,3 +1,4 @@
+/** @format */
 /**
```

Comment:
I don't think this is a good PR to `format` this file in

Reply:
👍 I hope for a near future where the whole codebase is already pretty!

In the meantime, I'll spin up a separate PR for that.